components/bio-formats-tools/test/loci/formats/tools/ImageConverterTest.java

```diff
@@ -181,4 +182,24 @@ public void testBadArgument() throws FormatException, IOException {
         System.getProperty("line.separator"), outContent.toString());
     }
   }
+
+  @Test
+  public void testCompanion() throws FormatException, IOException {
+    outFile = File.createTempFile("test", ".ome.tif");
+    outFile.delete();
+    File compFile = new File(outFile.getParentFile(), "test.companion.ome");
+    String[] args = {
+      "-option", OMETiffWriter.COMPANION_KEY, compFile.getAbsolutePath(),
+      "test.fake", outFile.getAbsolutePath()
+    };
+    try {
+      ImageConverter.main(args);
```

Comment:
Agreed. Not a blocker for this work at all. But since it's the second time someone in the team is confused by my approach, happy to move to using tempdir in `ImageConverterTest` as a follow-up PR if that improves the clarity.